### PR TITLE
Support multiple users, have 2nd user join created room

### PR DESCRIPTION
**Pull request by bwindels**
Implements these tests:
```
running tests ...
 * alice signs up ... done
 * alice accepts "Server Notices" invite and accepting terms & conditions ... done
 * bob signs up ... done
 * bob accepts "Server Notices" invite and accepting terms & conditions ... done
 creating a public room and join through directory:
 * alice creates room test ... done
 * alice changes the room settings:
   * alice sets directory listing to true ... done
   * alice sets visibility to public_no_guests ... done
 * bob joins room test ... done
 * bob writes "hi Alice!" in room ... done
 * alice receives message "hi Alice!" from bob in room ... done
all tests finished successfully
```

### CODE REVIEW

File: src/scenario.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  const room = 'test';
+  const message = "hi Alice!";
+  await createRoom(alice, room);
+  await changeRoomSettings(alice, {directory: true, visibility: "public_no_guests"});
+  await join(bob, room);
+  await sendMessage(bob, message);
+  await receiveMessage(alice, {sender: "bob", body: message});
+} 
+
+async function createE2ERoomAndTalk(alice, bob) {
```
Comment (dbkr) [inline src/scenario.js]: doesn't look like this is used anywhere?

Comment (dbkr) [reply src/scenario.js]: ah, you use it in the next PR

**Comment by bwindels** [comment]
@dbkr No reason apart from that what I'm used to and reverted to without realizing when starting this project. I'm happy to change it to 4 spaces to make it more in line with the other projects, but let's do it in a separate PR.

**Comment by bwindels** [comment]
https://github.com/matrix-org/matrix-react-end-to-end-tests/issues/7

**Pull request merged**